components/SwipeRow.js

```diff
@@ -134,24 +140,35 @@ class SwipeRow extends Component {
 		}
 	}
 
-	handlePanResponderEnd(e, gestureState) {
-		// re-enable scrolling on listView parent
+	ensureScrollEnabled() {
```

Comment:
Can we use auto-binding here. i.e:

```
ensureScrollEnabled = () => {
```

And then we can remove line 44 in the `constructor`

Reply:
Ah nice, didn't realise I could do that in a class definition!